packages/perseus/src/widgets/measurer/measurer.tsx

```diff
@@ -21,51 +17,62 @@ const defaultImage = {
     left: 0,
 } as const;
 
-const Measurer: any = createReactClass({
-    displayName: "Measurer",
-
-    propTypes: {
-        apiOptions: ApiOptions.propTypes,
-        box: PropTypes.arrayOf(PropTypes.number),
-        image: PropTypes.shape({
-            url: PropTypes.string,
-            top: PropTypes.number,
-            left: PropTypes.number,
-        }),
-        showProtractor: PropTypes.bool,
-        protractorX: PropTypes.number,
-        protractorY: PropTypes.number,
-        showRuler: PropTypes.bool,
-        rulerLabel: PropTypes.string,
-        rulerTicks: PropTypes.number,
-        rulerPixels: PropTypes.number,
-        rulerLength: PropTypes.number,
-    },
-
-    getDefaultProps: function () {
-        return {
-            box: [480, 480],
-            image: {},
-            showProtractor: true,
-            protractorX: 7.5,
-            protractorY: 0.5,
-            showRuler: false,
-            rulerLabel: "",
-            rulerTicks: 10,
-            rulerPixels: 40,
-            rulerLength: 10,
-        };
-    },
-
-    getInitialState: function () {
-        return {};
-    },
-
-    componentDidMount: function () {
+type Props = {
```

Comment:
This is just a standard migration from `createReactClass()` to a true `React.Component` class.

Reply:
Just FYI:
- https://khanacademy.atlassian.net/browse/LEMS-365
- https://khanacademy.atlassian.net/browse/LEMS-2393

Reply:
Thanks Matthew. I should have commented further... I needed to do the transform so that I could make a class that conforms to the Widget interface. I'd not have touched it otherwise. :)

Reply:
I'll mark 2393 as done when I get this deployed.